AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/InitialSync/InitialSyncOperationTests.swift

```diff
@@ -731,4 +731,80 @@ class InitialSyncOperationTests: XCTestCase {
         waitForExpectations(timeout: 1)
         sink.cancel()
     }
+
+    func testBaseQueryWithSyncExpressionConstantAll() throws {
+        let storageAdapter = try SQLiteStorageEngineAdapter(connection: Connection(.inMemory))
+        try storageAdapter.setUp(modelSchemas: StorageEngine.systemModelSchemas + [MockSynced.schema])
+
+        let apiWasQueried = expectation(description: "API was queried with sync expression")
+        let responder = QueryRequestListenerResponder<PaginatedList<AnyModel>> { request, listener in
+            XCTAssertEqual(request.document, """
+            query SyncMockSynceds($filter: ModelMockSyncedFilterInput, $limit: Int) {
+              syncMockSynceds(filter: $filter, limit: $limit) {
+                items {
+                  id
+                  __typename
+                  _version
+                  _deleted
+                  _lastChangedAt
+                }
+                nextToken
+                startedAt
+              }
+            }
+            """)
+            guard let filter = request.variables?["filter"] as? [String: Any?] else {
+                XCTFail("Unable to get filter")
+                return nil
+            }
+            XCTAssert(filter.isEmpty)
```

Comment:
i don't think this is the assert scenario anymore, since the filter doesn't get added, so `request.variables["filter"]` should be nil